icgcget/clients/commands/command_utils.py

```diff
@@ -0,0 +1,34 @@
+import click
+from ..errors import ApiError
```

Comment:
Yes.  For whatever reason, python can't make a relative import that goes to the directory of the main method.